ingress/controllers/nginx/nginx/main.go

```diff
@@ -67,7 +67,9 @@ func NewManager(kubeClient *client.Client) *Manager {
 
 	ngx.sslDHParam = ngx.SearchDHParamFile(config.SSLDirectory)
 
-	ngx.loadTemplate()
+	if err := ngx.loadTemplate(); err != nil {
+		glog.Fatalf("invalid NGINX template: %v", err)
```

Comment:
I'm specifically asking about what we should do if someone modifies the template to have a syntax error. I think the answer is we should do whatever nginx would do if someone modified the nginx.conf to have a syntax error.